### ISSUE TITLE
Aether II Mac Texture Glitch

### ISSUE THREAD

**Issue by Popcorn3568** [gildedgames/aether-issues#627]
Hi, I recently downloaded Aether II. I downloaded Forge, Guilded Games, and the Aether mod. I followed all the necessary steps to download the mod, and was able to access the mod and play it. However, something is wrong with the textures. I have attached pictures of what I am seeing. It is seemingly random, but about half of the textures of items in the game are white, pixellated squares. The armor slots are also pixellated like this. When I mouse over the item, it will still show the item name, but it is nearly impossible to play due to the fact that I don't know what any of the items are. I have done a lot of research and found a similar article that had to do with texture glitches in Aether II, but the instructions were not clear as to how to fix the problem. Does anyone know what I can do to fix this? Thanks so much in advance!

<img width="1440" alt="screen shot 2016-03-10 at 11 30 20 pm" src="https://cloud.githubusercontent.com/assets/17776878/13693473/9c4653b4-e719-11e5-8b4a-47d5bed86a7f.png">
<img width="1440" alt="screen shot 2016-03-10 at 11 30 24 pm" src="https://cloud.githubusercontent.com/assets/17776878/13693475/9c60fe3a-e719-11e5-8668-098e17ea8467.png">
<img width="1440" alt="screen shot 2016-03-10 at 11 30 26 pm" src="https://cloud.githubusercontent.com/assets/17776878/13693478/9c717ac6-e719-11e5-9273-eb7c82c997eb.png">
<img width="1440" alt="screen shot 2016-03-10 at 11 30 28 pm" src="https://cloud.githubusercontent.com/assets/17776878/13693476/9c657320-e719-11e5-8c5b-075322f71402.png">
<img width="1440" alt="screen shot 2016-03-10 at 11 30 31 pm" src="https://cloud.githubusercontent.com/assets/17776878/13693477/9c6b4d4a-e719-11e5-94be-8db55f5b5bfd.png">
<img width="1440" alt="screen shot 2016-03-10 at 11 30 34 pm" src="https://cloud.githubusercontent.com/assets/17776878/13693474/9c5abfc0-e719-11e5-9a51-36e90d439f43.png">

**Comment by Tamaized** [comment]
Sigh.. #579

**Comment by Mooplez** [comment]
I tried to help her with this in real life. We found that thread and didn't understand it exactly. We followed what Collin said but we need a better description of the whole splash.properties part. What does he mean by create a file? What type of file and how? I use pc so all this was clueless to me and his fix was pretty vague from what I could tell. We tried to make a text file but that didn't work so something was obviously not right.
That would be the similar thread she referred to fyi.

**Comment by Tamaized** [comment]
It means create a file named "splash" with the file extension "properties" (Hence the dot in the middle of it) and the contents of this file is "enabled=false". 

As for the location of this file, i'm pretty sure its "/Users/USERNAME/Library/Application Support/.minecraft/config" According to Collin. (Though this location depends on where you have things set but this is default)

**Comment by Popcorn3568** [comment]
Thank you, I tried this, and it did not affect the textures. I created a text file, followed the instructions, and made sure the extension was .properties, but the textures were still bugged out. Do you know if there is anything else I can try? Maybe another tutorial?

**Comment by Tamaized** [comment]
As far as I know, what this is supposed to do is disable MinecraftForge's new load screen which in return fixes this issue. (The one with the spinning hammer and anvil when you first load the game up, or not spinning, that thing has changed a few times and I hardly remember what it looks like now)

If that load screen isn't being disabled by this then something was done wrong, if it is being disabled and this issue still happens then I wouldn't know the cause or how to fix it.

**Comment by Popcorn3568** [comment]
Yeah, the load screen still shows up when I try to open the game. I'll do some more research on how to disable that and see what I can do. Thanks for all of your help.

**Comment by Popcorn3568** [comment]
I was able to find the issue. A new splash.properties file was created in the config files that I did not create. I changed this into a text file so that I could edit it, and there was a separate enabled that was set to true. I changed this to false, changed the splash file back to a .properties file, and the load screen no longer appeared when I opened Aether. The textures are now fixed. Thank you so much for all of your help!
